app/assets/stylesheets/me-add-to-playlist.css.scss.erb

```diff
@@ -1,7 +1,49 @@
 .scrubber-marker {
   width: 1em !important;
   bottom: 0;
-  transform: translate(-50%,100%)
+  transform: translate(-50%,200%)
+}
+
+.mejs-time-marker-rail {
+  margin: 5px;
+}
+
+.mejs-time-float-marker {
+  background: #eee;
+  height: auto !important;
+  width: 100px !important;
+  border: solid 1px #333;
+  top: auto;
+  bottom: 8px;
+  transform: translate(-50%, 0);
+  text-align: center;
+  color: #111;
+  z-index: 1;
```

Comment:
As long as this z-index property is doing what it's intended, all good.  Generally for a z-index to take effect the class would need a `position` property, but perhaps it's inherited.